src/app/beer_garden/command_publishing_blocklist.py

```diff
@@ -0,0 +1,22 @@
+from beer_garden.db.mongo.models import CommandPublishingBlockList
+
+
+def command_publishing_block_list_adds(command):
+    blocked_command = CommandPublishingBlockList(**command)
+    blocked_command.save()
+    return blocked_command
+
+
+def command_publishing_block_list_remove(command_publishing_id):
+    blocked_command = CommandPublishingBlockList.objects.get(id=command_publishing_id)
+    blocked_command.delete()
+
+
+def command_publishing_block_list_get():
```

Comment:
This likely goes away in favor of using the schema to generate the appropriate response.